HerPublicWebsite.BusinessLogic/Models/AnonymisedReport.cs

```diff
@@ -0,0 +1,35 @@
+﻿using HerPublicWebsite.BusinessLogic.Models.Enums;
+
+using HerPublicWebsite.BusinessLogic.Extensions;
+
+namespace HerPublicWebsite.BusinessLogic.Models;
+
+public class AnonymisedReport
+{
+    public string PostcodeFirstHalf { get; set; }
+    public bool IsLsoaProperty { get; set; }
+    public EpcRating EpcRating { get; set; }
+    public DateTime? EpcLodgementDate { get; set; }
+    public bool IsEligible { get; set; }
+    public HasGasBoiler HasGasBoiler { get; set; }
+    public IncomeBand IncomeBand { get; set; }
+    public string CustodianCode { get; set; }
+    public OwnershipStatus OwnershipStatus { get; set; }
+
+    public AnonymisedReport()
+    {
+    }
+
+    public AnonymisedReport(Questionnaire questionnaire)
+    {
+        PostcodeFirstHalf = questionnaire.AddressPostcode.NormaliseToUkPostcodeFormat().Split(" ").FirstOrDefault();
```

Comment:
This is just about complicated enough, and definitely important enough for GDPR to have a couple of tests around it.